internal/renderer.go

```diff
@@ -19,6 +19,7 @@ func renderResources(cfg *SiteConfig, version string) (string, error) {
 		Funcs(helpers.TemplateFuncs()).
 		Funcs(map[string]any{
 			"RenderScopes": renderScope,
+			"derefBool": func(i *bool, defaultValue bool) bool { if i == nil{return defaultValue } else{ return *i} },
```

Comment:
let's remove support for defaultValue since now we have defaults defined on two places (here and in the struct). If the pointer is nil then it should return false